### PR TITLE
ui bug fix: scalevm is disabled when vm is Stopped

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes the issue on ui that scalevm button is disabled when vm is stopped.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [X] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):

prior this fix

image

with this fix

image

### CODE REVIEW

File: ui/src/config/section/compute.js [hunk 1]
```diff
@@ -299,7 +299,7 @@ export default {
           docHelp: 'adminguide/virtual_machines.html#how-to-dynamically-scale-cpu-and-ram',
           dataView: true,
           show: (record) => { return ['Stopped'].includes(record.state) || (['Running'].includes(record.state) && record.hypervisor !== 'KVM' && record.hypervisor !== 'LXC') },
-          disabled: (record) => { return !record.isdynamicallyscalable },
+          disabled: (record) => { return !record.isdynamicallyscalable && record.state !== 'Stopped' },
```
Comment (shwstppr) [inline ui/src/config/section/compute.js:302]: Why not `disabled: (record) => { return record.state === 'Running' && !record.isdynamicallyscalable }`

Comment (ravening) [reply ui/src/config/section/compute.js:302]: > Maybe
> `disabled: (record) => { return record.state === 'Running' && !record.isdynamicallyscalable }`

but isnt `record.state !== 'Stopped'` and `record.state === 'Running' ` are same?

Comment (shwstppr) [reply ui/src/config/section/compute.js:302]: @ravening In this case yes. My point was to check VM's state first and isdynamicallyscalable later as that is relevant only for running VMs.

Comment (shwstppr) [reply ui/src/config/section/compute.js:302]: Anyway, I'm fine with original change as well

**Comment by shwstppr** [comment]
@blueorangutan ui

File: ui/src/config/section/compute.js (hunk 1 above)
Comment (weizhouapache) [reply ui/src/config/section/compute.js]: updated. thanks @shwstppr @ravening

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]
LGTM

Before changes (Scale VM disabled for a stopped VM):
Screenshot from 2021-07-22 17-15-36

After changes (Scale VM enabled for a stopped VM):
Screenshot from 2021-07-22 17-16-11

**Comment by weizhouapache** [comment]
> LGTM
> 
> Before changes (Scale VM disabled for a stopped VM):
> Screenshot from 2021-07-22 17-15-36
> 
> After changes (Scale VM enabled for a stopped VM):
> Screenshot from 2021-07-22 17-16-11

good, thanks for testing @shwstppr

**Review by harikrishna-patnala — APPROVED** [review]
LGTM. thanks for fixing @weizhouapache

**Review by davidjumani — APPROVED** [review]

**Pull request merged**